modulefiles/orion.intel19

```diff
@@ -2,10 +2,9 @@
 ##      GODAS prerequisites
 ##
 
-export OPT=/home/rmahajan/opt
```

Comment:
Why do we delete this line?

Reply:
@flampouris /modulefile is the wrong place to put this kind of variables.